src/renderers/dom/ReactDOM.js

```diff
@@ -59,15 +59,23 @@ if (__DEV__) {
   var ExecutionEnvironment = require('ExecutionEnvironment');
   if (ExecutionEnvironment.canUseDOM && window.top === window.self) {
 
-    // If we're in Chrome, look for the devtools marker and provide a download
-    // link if not installed.
-    if (navigator.userAgent.indexOf('Chrome') > -1) {
-      if (typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ === 'undefined') {
+    // First check if devtools is not installed
+    if (typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ === 'undefined') {
+      // If we're in Chrome or Firefox, provide a download link if not installed.
+      if (navigator.userAgent.indexOf('Chrome') > -1 &&
+          navigator.userAgent.indexOf('Edge') === -1) {
         console.debug(
           'Download the React DevTools for a better development experience: ' +
           'https://fb.me/react-devtools'
         );
       }
+
+      if (navigator.userAgent.indexOf('Firefox') > -1) {
+        console.debug(
+          'Download the React DevTools for a better development experience: ' +
+          'https://github.com/facebook/react-devtools/releases'
```

Comment:
We can probably just make this `fb.me/react-devtools-firefox` (and then change the Chrome one to `fb.me/react-devtools-chrome`.

Maybe eventually we'll add a dedicated devtools section of the site and we can just point everybody there.